xsdata/formats/dataclass/serializers/json.py

```diff
@@ -0,0 +1,62 @@
+import json
+from dataclasses import asdict
+from dataclasses import dataclass
+from dataclasses import field
+from decimal import Decimal
+from enum import Enum
+from typing import Callable
+from typing import Dict
+from typing import Optional
+from typing import Tuple
+from typing import Type
+
+from xsdata.formats.mixins import AbstractSerializer
+
+
+def filter_none(x: Tuple):
+    return dict((k, v) for k, v in x if v is not None)
+
+
+class DictFactory:
+    FILTER_NONE = filter_none
+
+
+class JsonEncoder(json.JSONEncoder):
+    def default(self, obj):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [An attribute defined in json.encoder line 158 hides this method](https://app.codacy.com/manual/tefra/xsdata/pullRequest?prid=5130997)